src/realm/group.hpp

```diff
@@ -788,14 +788,16 @@ class Group : public ArrayParent {
     ///
     ///  23 Layout of Set and Dictionary changed.
     ///
+    ///  24 Variable sized arrays for Decimal128.
```

Comment:
Should we mention lack of upgrade support for pre v10 here as well?

Reply:
That is not a quality of the file format version.